src/go/k8s/tests/e2e-v2-ci/create-kuttl-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+mkdir -p tests
+
+# create the directory structure
+helm template redpanda operator-tests -s templates/rp.yaml | yq '"tests/" + .metadata.name' | xargs mkdir -p
```

Comment:
why `helm template`, but not https://github.com/hairyhenderson/gomplate or https://coveooss.github.io/gotemplate/?